binary/util.py

```diff
@@ -487,6 +487,9 @@ def encoder_for(self, param_type):
     'DiskTierConfig': 'aDiskTierConfig',
     'TieredStoreConfig': 'aTieredStoreConfig',
     'List_PartitioningAttributeConfig': 'aListOfPartitioningAttributeConfigs',
+    'WanConsumerConfigHolder': 'aWanConsumerConfigHolder',
+    'List_WanCustomPublisherConfigHolder': 'aListOfWanCustomPublisherConfigsHolders',
+    'List_WanBatchPublisherConfigHolder': 'aListOfWanBatchPublisherConfigHolders',
```

Comment:
Do you know why we don't need to add an instance here for WanSyncConfig, DiscoveryConfig etc. 🤔 ?

Reply:
They aren't top-level reference in the `addWanReplicationConfig` API. The `ReferenceObjects` in OS project defines the nested instances.